addons/account/models/res_users.py

```diff
@@ -7,6 +7,16 @@
 class ResUsers(models.Model):
     _inherit = 'res.users'
 
+    has_group_show_price_subtotal = fields.Boolean(
```

Comment:
These fields are already defined in sale/models.res_users.py. 

You also ovewrite existing fields..